src/client/storage/index.ts

```diff
@@ -337,7 +162,7 @@ const uploadDir = async dirPath => {
     const files = await fs.readdir(dirPath);
     const dirInfo = {
         type: 'dir',
-        files: []
+        files: [] as any[] // TODO: any
```

Comment:
Let's define the file type right away, it seems to be already clear form the below structure (line 176).